.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Documentation request
+about: Missing documentation or suggest an improvement
+
+---
+
+<!--
+Thanks for submitting this documentation request. Please fill the template
+below, otherwise we will not be able to process this request.
+
+Open an issue with this template if you think documentation regarding a certain
```

Comment:
We don't have such an explanation in our other templates. I think the `about` in the metadata should be sufficient. Also keep in mind that people only see this if they already selected "Documentation request"